extensions/amp-video-docking/0.1/controls.js

```diff
@@ -395,7 +458,9 @@ export class Controls {
    * @param {number} height
    */
   positionOnVsync(scale, x, y, width, height) {
-    const {container, dismissContainer_: dismissContainer} = this;
+    this.area_ = layoutRectLtwh(x, y, width, height);
```

Comment:
`mouseout` evaluates coordinates based on bounding box, so when the docked box changed, controls would break badly. Updating the locally known `area_` fixes it.